README.md

```diff
@@ -32,40 +38,47 @@ Heroku CNBs are written with the app developer in mind. We want it to be as easy
 
 #### pack
 
-To run our buildpack locally, we will use `pack`, a tool maintained by the Cloud Native Buildpacks project to support the use of buildpacks. Find their documentation about installing it here: https://buildpacks.io/docs/install-pack/
+To run our buildpack locally, we will use `pack`, a tool maintained by the Cloud Native Buildpacks project. Follow their documentation to [install the pack CLI](https://buildpacks.io/docs/install-pack/).
 
 ### Create the app image
 
-First, we need an app to transform into an image. We use [Heroku's "getting started" app for Node.js](https://github.com/heroku/node-js-getting-started), but any other app that uses a supported ecosystem will work. We can clone the repository like so: `git clone https://github.com/heroku/node-js-getting-started.git`. 
+> [!NOTE]
+> For a full hands-on getting started experience [follow our getting started tutorial](TUTORIAL.md).
 
-Then, we can run the following command to build our app image - no additional steps needed! The required buildpacks for your app will be detected automatically.
+First, we need an application to produce an image. We will use [Heroku's getting started app for Node.js](https://github.com/heroku/node-js-getting-started), but any other app that uses a supported ecosystem will work. Clone the application:
 
 ```
-pack build my-app-image --builder heroku/builder:22 --path node-js-getting-started
+$ git clone https://github.com/heroku/node-js-getting-started.git
+$ cd node-js-getting-started
 ```
 
-### Tutorial
+In your application directory, run the `pack build` command. No additional steps are needed! The required buildpacks for your app will be detected automatically:
 
-Check out our [hands-on tutorial](TUTORIAL.md) to learn more about how our buildpacks can help you create production-ready container images.
+```
+$ pack build my-app-image --builder heroku/builder:22 --path .
+```
+
+Once the image is built, you can run it with the tools you're already comfortable with, like the `docker` CLI, for a demonstration of how [read out getting started tutorial](TUTORIAL.md).
 
 ## Discussions
 
-Overarching discussions about the Heroku Buildpacks project happen on GitHub in the [discussions tab of this repository](https://github.com/heroku/buildpacks/discussions). Feel free to open a new discussion if you have any questions, feedback or other input - we want to hear from you!
+We want to provide great user experiences and can only do that with your help. Please try our Cloud Native Buildpacks and let us know how it went. What went well? What could be better?  Let us know on GitHub in the [discussions tab of this repository](https://github.com/heroku/buildpacks/discussions). Feel free to open a new discussion if you have any questions, feedback, or other input - we want to hear from you!
```

Comment:
👍

Reply:
When I read the PR that introduced "Overarching", I had to look it up to verify it was a valid English word :) Turns out it is, but this is a better way to phrase the sentiment.

Think we can probably drop the last sentence/"call to action" now though (seeing as we've already solicited feedback/discussions)? 

```suggestion
We want to provide great user experiences and can only do that with your help. Please try our Cloud Native Buildpacks and let us know how it went. What went well? What could be better?  Let us know on GitHub in the [discussions tab of this repository](https://github.com/heroku/buildpacks/discussions)!
```